PERFORMANCE

```diff
@@ -37,6 +37,55 @@ To characterize Chapel performance, generally speaking...
   assignments are used to move chunks of data between locales.
 
 
+Experimental flags for improving performance
+--------------------------------------------
+
+The following compiler flags are intended to provide a preview of
+performance improvements that we anticipate being delivered by the
+compiler automatically as it matures.  Both are available for use "at
+your own risk" in that they are not guaranteed to maintain program
+correctness (details available after each flag).
+
+* chpl -sassertNoSlicing=true ...
+
+  Chapel's arrays are currently heavyweight compared to C/Fortran in
```

Comment:
I think the first sentence might paint a worse picture than it needs to. In particular at first glance I read it as all aspects of Chapel's arrays are heavyweight. 

Maybe: 'Indexing into Chapel's arrays' and maybe change "heavyweight" to something with a kinder connotation?